containers/dotnetcore-mssql/.devcontainer/mssql/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM mcr.microsoft.com/mssql/server:2019-latest
+
+USER root
+
+COPY setup.sql *.dacpac ./
```

Comment:
The challenge here is this will all get backed into the image - as I mentioned in setup-db.sh, could this be done via postCreateCommand instead?